client/src/App.tsx

```diff
@@ -62,6 +62,7 @@ const App = () => {
               />
             }
           />
+          {/* let url = `multiple-results/${latitude}/${longitude}////${radius}/${open_now}`; */}
```

Comment:
I forgot to remove this comment, it will be removed before I merge.

Reply:
noted.